documentation/src/main/wiki/HandlebarsKnot.md

```diff
@@ -76,7 +76,8 @@ Main Handlebars Knot options available.
 | Name                        | Type                                | Mandatory      | Description  |
 |-------:                     |:-------:                            |:-------:       |-------|
 | `address`                   | `String`                            | &#10004;       | Event bus address of the Handlebars Knot verticle. |
-| `templateDebug`             | `Boolean`                           | &#10004;       | Template debug enabled option.|
+| `cacheSize`                 | `Long`                              | &#10004;       | Max cache size for compiled Handlebars snippets.|
```

Comment:
Looks that it is not mandatory. Default value is `1000`.

Reply:
yes, fixed.
I remove this entry from default configuration too.